contracts/index_executor/src/task_deposit.rs

```diff
@@ -110,13 +110,15 @@ impl DepositData {
         worker: [u8; 32],
     ) -> Result<Task, &'static str> {
         pink_extension::debug!("Trying to parse task data from json string");
-        let execution_plan_json: ExecutionPlan =
-            pink_json::from_str(&self.task).map_err(|_| "InvalidTask")?;
+
+        let execution_plan: ExecutionPlan =
+            Decode::decode(&mut &hex::decode(&self.task).expect("InvalidTask")[..])
```

Comment:
Since we changed solution type from `string` to `bytes` in solidity, here in both `DepositData`, `EvmDepositData` and `SubDepositData` (pallet-index already use Vec<u8>) should change type of task to `Vec<u8>`, and decode like below:
```suggestion
            let mut task = self.task;
            Decode::decode(&mut task).map_err(|_| Err("InvalidTask"))
```